guide/how-it-works/lightning-service-providers.md

```diff
@@ -0,0 +1,310 @@
+how-it-works/lightning-service-providers---
+layout: guide
+title: Lightning Service Providers
+description: Common user experience problems of the lightning network that can be resolved by third-party services.
+nav_order: 3
+parent: How it works
+permalink: /guide/how-it-works/lightning-service-providers/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=701%3A5911
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-service-providers/lightning-service-providers.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-service-providers/lightning-service-providers@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/lightning-service-providers/lightning-service-providers-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-service-providers/lightning-service-providers-mobile@2x.jpg"
+   alt-text = "Lightning service providers header image."
+   width = 1600
+   height = 740
+   layout = "full-width"
+%}
+
+# Lightning Service Providers
+
+The [lightning network]({{ '/guide/getting-started/technology-primer/#the-lightning-payment-network/' | relative_url }}) is currently bitcoins best solution at [scaling]({{ '/guide/getting-started/technology-primer/#do-all-transactions-have-to-be-this-secure/' | relative_url }}) to billions of people. However, using the lightning network self-custodially brings about many new usability issues, such as channel management, routing, uptime requirements and backups.
+
+Lightning service providers (LSPs) solve these user friction points by providing services that make it easy to access and use the lightning network. They act as trust-minimized third parties that make it easy to onboard new user into lightning.
+
+Roy Sheinfeld from [Breez](https://breez.technology/) explains LSPs in more depth in his [introducing lightning service providers](https://medium.com/breez-technology/introducing-lightning-service-providers-fe9fb1665d5f) and [envisioning LSPs in the lightning economy](https://medium.com/breez-technology/envisioning-lsps-in-the-lightning-economy-832b45871992) articles.
+
+## What is a lightning service?
+
+Lightning services are trust-minimized offerings by LSPs that solve user pain points when using a non-custodial bitcoin lightning wallet.
+
+A lightning service should always non-custodial, be opt-out, only used when necessary and any trade-offs should be clearly communicated to users.
```

Comment:
This sentence is a bit hard to read. Was this the point of the sentence?
```suggestion
A lightning service should always be non-custodial, opt-out, with trade-offs clearly communicated to users.
```

Reply:
```suggestion
A lightning service should always be non-custodial and opt-out, with trade-offs clearly communicated to users.
```
I'd like to take the suggestion from @pavlenex a step further. "And" is better than a comma in this context.